concepts/zero-values/about.md

```diff
@@ -1,30 +1,67 @@
 # Zero Values
 
-Unlike some other languages, Go has no empty/null/undefined state for uninitialized variables. Every type has a [zero value][zero_values] that uninitialized variables will hold upon declaration.
+Go does not have a concept of undefined for variable values. Variables declared without an explicit initial value default to the [zero value][zero_values] for their respective type.
 
-These default values are called the zero values for their respective types:
+The zero value for primative types such as booleans, numeric types, and strings are `false`, `0`, and `""`, respectively.
+
+The identifier `nil`, meaning zero, is the zero value for more complex types such as pointers, interfaces, channels, maps, slices, and functions.
```

Comment:
I would argue that this document does not need to describe `nil` values (as that is a different concept). If you'd like to link between concepts, I'd suggest using the Concept Widget as described in this document.

Reply:
Removed the `nil` values and added a Concept Widget.